### ISSUE TITLE
Sort packages in lexicographic order

### ISSUE THREAD

**Issue by samoht** [ocamllabs/opam-doc#93]
The list in http://mirage.github.io/ looks a little bit too random to me (well, I guess it reflects some kind of build dependency order, but that's far from ideal)

**Comment by avsm** [comment]
Agreed.  Note that I want to rename the existing front page to /pkg, and have the option for a front page template that can be hand written to hide the intermediate packages as much as possible.

**Comment by dsheets** [comment]
I think we should move to putting these under `/packages` which means that opam2web and the website addresses will have to change as well. This is just an attempt to align everything with the layout of an opam repo.
